azure/src/services/azureFunctionCloudService.ts

```diff
@@ -53,20 +54,10 @@ export class AzureFunctionCloudService implements CloudService {
   public containerResolver: ContainerResolver;
 
   /**
-   *
-   * @param name Name of function to invoke
-   * @param fireAndForget Wait for response if false (default behavior)
-   * @param payload Body of HTTP request
-   * @param headers Headers of the context
-   * @param params StringParams with values for URL
+   * @param params invoke interface with parameters needed
    */
-  public async invoke<T>(
-    name: string,
-    fireAndForget,
-    payload: any = null,
-    headers: StringParams = new StringParams(),
-    params: StringParams = new StringParams()
-  ) {
+  public async invoke<T>( params: InvokeRequest) {
```

Comment:
nit: remove extra space between `parenthesis` and `params`

Reply:
Added this change in my next commit. Thanks for the feedback